adapters/risc0/src/guest.rs

```diff
@@ -40,4 +41,14 @@ impl Zkvm for Risc0Guest {
         // Implement this method once risc0 supports recursion
         todo!()
     }
+
+    fn verify_and_extract_output<
+        C: sov_rollup_interface::zk::ValidityCondition,
+        Add: AddressTrait,
+    >(
+        _serialized_proof: &[u8],
+        _code_commitment: &Self::CodeCommitment,
+    ) -> Result<sov_rollup_interface::zk::StateTransition<C, Add>, Self::Error> {
+        todo!()
```

Comment:
Why is it not implemented? We can put text inside todo, that will indicate why, and when it is going to be implmeneted